wallets/wrap.js

```diff
@@ -0,0 +1,154 @@
+import { createHodlInvoice, parsePaymentRequest } from 'ln-service'
+import { estimateRouteFee } from '../api/lnd'
+
+const MIN_OUTGOING_MSATS = BigInt(900) // the minimum msats we'll allow for the outgoing invoice
+const MAX_OUTGOING_MSATS = BigInt(9_000_000_000) // the maximum msats we'll allow for the outgoing  invoice
+const MAX_EXPIRATION_INCOMING_MSECS = 900_000 // the maximum expiration time we'll allow for the incoming invoice
+const INCOMING_EXPIRATION_BUFFER_MSECS = 300_000 // the buffer enforce for the incoming invoice expiration
+const MIN_INCOMING_CLTV_DELTA = 200 // the minimum cltv delta we'll allow for the incoming invoice
+const MAX_OUTGOING_CLTV_DELTA = 1500 // the maximum cltv delta we'll allow for the outgoing invoice
```

Comment:
I don't think this is the value we want in prod (ideally <600), but for some reason in dev our estimation of this is way bigger than it should be. More research needed.

Reply:
This is resolved in [009ae43](https://github.com/stackernews/stacker.news/pull/1178/commits/009ae4332d516afd3015aa82db8b71b6c8bb4993).

`ln-service` expects absolute cltv heights, so we need to factor in current block height. I had assumed they were deltas.